packages/extension-store-locator/README.md

```diff
@@ -0,0 +1,92 @@
+# Store Locator Extension
+
+A PWA Kit extension that adds store locator functionality to your application. This extension provides a store locator solution with features like:
+
+- Search stores by postal code / country code via SCAPI Shopper Stores API
+- Device geolocation support
+- Configurable search radius
+- Multi-country support
+
+## Installation
+
+```sh
+npm install @salesforce/extension-store-locator
+```
+
+## Peer Dependancies
+
+PWA-Kit Application Extensions are NPM packages at their most simplest form, and as such you can define
+what peer dependencies are required when using it. Because this application extension provides
+Chakra UI via a page and components, it requires that the some peer dependencies are installed.
+
+Depending on what features your application extensions provides it's recommended you include any third-party
+packages as peer dependencies so that your base application doesn't end up having multiple versions of a 
+given package. See package.json for the full list of peer dependencies.
+
+## Configurations
+
+The Store Locator extension is configured via the `mobify.app.extensions` property in the config files or `package.json` file.
+
+```json
+{
+  "mobify": {
+    "app": {
+      "extensions": [
+        [
+          "@salesforce/extension-store-locator",
+          {
+            "enabled": true,
+            "path": "/store-locator",
+            "defaultDistance": 100,
+            "defaultDistanceUnit": "km",
+            "defaultPageSize": 10,
+            "defaultPostalCode": "10178",
+            "defaultCountry": "Germany",
+            "defaultCountryCode": "DE",
+            "supportedCountries": [
+              {
+                "countryCode": "US",
+                "countryName": "United States"
+              },
+              {
+                "countryCode": "DE",
+                "countryName": "Germany"
+              }
+            ]
+          }
+        ]
+      ]
+    }
+  }
+}
+```
+
+
+## Overridable Files
+
+The following files in the extension can be overridden in your project:
+
+```
+/src/components/store-locator/store-locator-content.tsx
+/src/components/store-locator/store-locator-form.tsx
+/src/components/store-locator/store-locator-list.tsx
+/src/components/store-locator/store-locator-modal.tsx
+/src/components/store-locator/store-locator-provider.tsx
+/src/components/store-locator/use-store-locator.ts
+/src/components/store-locator/with-optional-chakra-provider.tsx
+/src/components/store-locator/with-store-locator.tsx
+/src/pages/store-locator/index.tsx
+/src/types/config.ts
+/src/types/index.ts
```

Comment:
I like that you took the time to refactor and list out these items. But, I think that we should experiment with offering a more limited set of overridable components. 

The one component I was expecting to see here that isn't, would be a list item. 

Lets work on coming up with the proper list of overridable components for this extension.

Reply:
What do you think about this list?

/src/pages/store-locator/index.tsx
/src/components/store-locator/store-locator-content.tsx
/src/components/store-locator/store-locator-form.tsx
/src/components/store-locator/store-locator-list.tsx
/src/components/store-locator/store-locator-list-item.tsx (to be created)
/src/components/store-locator/store-locator-modal.tsx

Reply:
Lets just make 1 things overridable:

- list item

If you are feeling saucy, maybe one for a "no stores found" component if that exists.